components/esb-tools/plugins/org.wso2.integrationstudio.artifact.synapse.api/src/org/wso2/integrationstudio/artifact/synapse/api/ui/wizard/SynapseAPICreationWizard.java

```diff
@@ -967,6 +1030,15 @@ public void openEditor(File file) {
             log.error("Cannot open the editor", e);
         }
     }
+    
+    private boolean isVersionTwoYamlFile(String yamlContent) {
+    	Pattern p = Pattern.compile("([ ]*)(swagger[ ]*:[ ]*[\\\"]?[2][.])(.*)");
+    	Matcher m = p.matcher(yamlContent.toLowerCase());
+    	if (m.find()) {
+    		return true;
```

Comment:
formatting issue

Reply:
This PR is reverted by: https://github.com/Tharsanan1/integration-studio/commit/ef1e82bf7abe20bfb8300d844d2533ca4ffbb6dd. Hence resolving the comment